src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -523,7 +553,7 @@ private int decodeHttpBegin(
         final int endOfHeadersAt = limitOfBytes(payload, offset, limit, ServerStreamFactory.CRLFCRLF_BYTES);
         if (endOfHeadersAt == -1)
         {
-            // Incomplete request, signal we can't consume the data
+            // Incomplete request, signal we can't setCleanupConnectReply the data
```

Comment:
Unintended search and replace change? Recommend using refactor to avoid this next time.